pkg/yurttunnel/server/interceptor_test.go

```diff
@@ -66,10 +66,10 @@ func TestGetResponse(t *testing.T) {
 		t.Errorf("get response failed")
 	}
 
-	if diffBytes(rbytes, wbytes) {
-		//t.Errorf("raw bytes is not equal\n")
-	}
-
+	//if diffBytes(rbytes, wbytes) {
+	//	t.Errorf("raw bytes is not equal\n")
+	//}
+	diffBytes(rbytes, wbytes)
```

Comment:
Will fail, I think should be changed to this:
```golang
if !diffBytes(rbytes, wbytes) {
    t.Errorf("raw bytes is not equal\n")
}
```
Because `diffBytes` in the case of equal return `true`.